.github/workflows/verify.yml

```diff
@@ -0,0 +1,33 @@
+name: Verify every day
+
+on:
+  schedule:
+    # every day at 9am
+    - cron: '0 9 * * *'
+
+env:
+  go-cache-name: go
```

Comment:
what's this used for?

Reply:
It's not being used in this workflow, so removed. Thanks